<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inputWindowClass</class>
 <widget class="QMainWindow" name="inputWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>输入界面</string>
  </property>
  <property name="windowIcon">
   <iconset resource="inputWindow.qrc">
    <normaloff>:/new/prefix1/image/input.png</normaloff>:/new/prefix1/image/input.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="noticeBox">
      <property name="title">
       <string>notice</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="noticeText">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="inputBox">
      <property name="title">
       <string>input</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTextEdit" name="inputText">
         <property name="placeholderText">
          <string>请输入...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="againBtn">
         <property name="text">
          <string>重写</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="confirmBtn">
         <property name="text">
          <string>确定</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="inputWindow.qrc"/>
 </resources>
 <connections/>
</ui>
